lib/rubocop/cop/rails/active_support_on_load.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Checks for Rails framework classes that are patched directly instead of using Active Support load hooks. Direct
+      # patching forcibly loads the framework referenced, using hooks defers loading until it's actually needed.
+      #
+      # @example
+      #
+      #   # bad
+      #   ActiveRecord::Base.include(MyClass)
+      #
+      #   # good
+      #   ActiveSupport.on_load(:active_record) { include MyClass }
+      #
+      # @safety
+      #   While using lazy load hooks is recommended, it changes the order in which is code is loaded and may reveal
+      #   load order dependency bugs.
+      class ActiveSupportOnLoad < Base
+        extend AutoCorrector
+
+        LOAD_HOOKS = {
```

Comment:
Can you move this `LOAD_HOOKS` constant under `RESTRICT_ON_SEND` constant?

Reply:
Yes, done.